specs-go/config.go

```diff
@@ -300,7 +300,7 @@ type LinuxCPU struct {
 // LinuxPids for Linux cgroup 'pids' resource management (Linux 4.3)
 type LinuxPids struct {
 	// Maximum number of PIDs. Default is "no limit".
-	Limit *int64 `json:"limit,omitempty"`
+	Limit int64 `json:"limit"`
```

Comment:
From the kernel side, `0` is functionally identical to `1` (because you can join a cgroup even if it has a limit of `0`).